### PR TITLE
I fixed all the issues. instead of node-cron thing i used await promise. isReady() function is to check if bot is ready or not.

### PR DESCRIPTION
**Pull request by KartikeSingh**

### CODE REVIEW

File: src/utility/sendCount.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        if (disableConsole) return;
+        console.log("[ botlists.js ] : Successfully Updated Server Count");
+    }).catch(() => {
+        if (!disableConsole) return;
+        console.log("[ botlists.js ] : Failed in Updating Server Count");
+    })
+
+    // Waiting for the timeout than re-sending the request
+    await new Promise((res) => setTimeout(res, timeout));
+    sendCount(client, token, timeout, disableConsole);
```
Comment (sell) [inline src/utility/sendCount.js:31]: No, don't do this. This isn't good. There also shouldn't be an option for the user to toggle the time. If they do they'll possibly get rate limited. So just leave it to run every hour, which is why a cron job would be nice.

File: src/utility/sendCount.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        status: client.user.presence ? client.user.presence.status : "online"
+    }, { // The authorization
+        headers: {
+            Authorization: token
+        }
+    }).then(() => {
+        if (disableConsole) return;
+        console.log("[ botlists.js ] : Successfully Updated Server Count");
+    }).catch(() => {
+        if (!disableConsole) return;
```
Comment (sell) [inline src/utility/sendCount.js:25]: Why is this one working if `disableConsole` is true?

File: src/utility/sendCount.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        guilds: client.guilds.cache.size,
+        shards: client.shard ? client.shard.count : 0,
+        status: client.user.presence ? client.user.presence.status : "online"
+    }, { // The authorization
+        headers: {
+            Authorization: token
+        }
+    }).then(() => {
+        if (disableConsole) return;
+        console.log("[ botlists.js ] : Successfully Updated Server Count");
```
Comment (sell) [inline src/utility/sendCount.js:23]:
```js
if (!disableConseol) {
  console.log("[ botlists.js ] : Successfully Updated Server Count");
}
```

In my opinion this is better as there is no need for that return statement.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+/**
+ * An module to auto send your server counts to botlists.com
+ * @param {*} client Your Discord Bot Client
+ * @param {String} token Your Bot's Token for Website API
+ * @param {Number} timeout Timeinterval between sending server count
+ * @param {Boolean} disableConsole Whether you want console logs or not
+ */
+module.exports = (client, token, timeout = 1800000, disableConsole = false) => {
+    return new Promise((resolve, reject) => {
```
Comment (sell) [inline src/index.js:11]: You're returning a promise but don't use it in the example.

**Review by sell — CHANGES_REQUESTED** [review]

File: src/utility/sendCount.js (hunk 2 above)
Comment (KartikeSingh) [reply src/utility/sendCount.js:25]: If disable console is true than return other wise don't return and log the thing

File: src/utility/sendCount.js (hunk 1 above)
Comment (KartikeSingh) [reply src/utility/sendCount.js:31]: Okay btw I made it than they minimum time should be 5 mins

File: src/utility/sendCount.js (hunk 3 above)
Comment (KartikeSingh) [reply src/utility/sendCount.js:23]: Same thing, will do like this

File: src/utility/sendCount.js (hunk 2 above)
Comment (KartikeSingh) [reply src/utility/sendCount.js:25]: oh lol i am stupid thats why

**Pull request closed without merging**